From 676452a92d1fc4f85e8d89930d7729197c031026 Mon Sep 17 00:00:00 2001
From: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
Date: Sat, 5 Oct 2013 15:51:56 +0200
Subject: [PATCH 1/3] libbb: use <poll.h> instead of <sys/poll.h>

As the pol() manpage recommends, include the <poll.h> header instead
of <sys/poll.h>. This allows to get rid of gazillions of warnings when
building Busybox against the musl C library, which prints a warning
when the internal header <sys/poll.h> is included directly instead of
the <poll.h> header.

Signed-off-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
---
 include/libbb.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/libbb.h b/include/libbb.h
index e520060..f03f454 100644
--- a/include/libbb.h
+++ b/include/libbb.h
@@ -37,7 +37,7 @@
 #include <libgen.h> /* dirname,basename */
 #undef basename
 #define basename dont_use_basename
-#include <sys/poll.h>
+#include <poll.h>
 #include <sys/ioctl.h>
 #include <sys/mman.h>
 #include <sys/socket.h>
-- 
1.8.1.2

